modules/oeFax/controller/FaxServer.php

```diff
@@ -0,0 +1,125 @@
+<?php
+/**
+ * Fax Server SMS Module Member
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Jerry Padgett <sjpadgett@gmail.com>
+ * @copyright Copyright (c) 2019 Jerry Padgett <sjpadgett@gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+$ignoreAuth = 1;
+require_once(__DIR__ . "/../../../interface/globals.php");
```

Comment:
Hopefully can remove the above 2 lines and include globals.php upstream when use the class. Also, doesn't this class need a namespace?

Reply:
It is a server app. It is for call backs from Twilio and never called from an app.

Reply:
To clarify, is it only called via this script (or are their other mechanisms):
modules/oeFax/faxserver/fax_serve.php